src/widgets/splits/Split.cpp

```diff
@@ -595,13 +649,7 @@ void Split::openWhispersInBrowser()
 
 void Split::openBrowserPlayer()
 {
-    ChannelPtr channel = this->getChannel();
-    if (auto twitchChannel = dynamic_cast<TwitchChannel *>(channel.get()))
-    {
-        QDesktopServices::openUrl(
-            "https://player.twitch.tv/?parent=twitch.tv&channel=" +
-            twitchChannel->getName());
-    }
+    this->openChannelInBrowserPlayer(this->getChannel());
```

Comment:
We could try to just replace (and potentially delete) `Split::openBrowserPlayer` with `Split::openChannelInBrowserPlayer` since we just have 2 functions that do literally the same, but maybe that's something that should be done in separate PR so just making a note for the future.

Reply:
Duplicated cause `Split::openBrowserPlayer` was a qt slot and meant for current Split while `Split::openChannelInBrowserPlayer` takes any channel. Same about `Split::openChannelInStreamlink`.

Reply:
> Duplicated cause Split::openBrowserPlayer was a qt slot and meant for current Split

Correct! Passing an empty channel or simply the current split's channel could be the way to go imho.

Reply:
> Correct! Passing an empty channel or simply the current split's channel could be the way to go imho.

I implemented it like this at first (with default channel arg = nulptr) but it wouldn't compile and later changed to current version. 
Something to do with slots. My C++ isn't very good :smile: